{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "links": {"attachments": {"href": "data/repositories/osrf/gazebo/issues/38/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/gazebo/issues/38.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/38/watch"}, "comments": {"href": "data/repositories/osrf/gazebo/issues/38/comments_page=1.json"}, "html": {"href": "#!/osrf/gazebo/issues/38/client-can-crash-leaving-server-still"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/38/vote"}}, "reporter": {"display_name": "Brian Gerkey", "uuid": "{3d8b5763-9dad-4be7-85c2-a7ab6cd49173}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D"}, "html": {"href": "https://bitbucket.org/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:2e35028e-9ad8-4d6a-a3a5-275159a304a9/5fa9ec73-0fec-4860-a202-e34ae23b6a12/128"}}, "nickname": "Brian Gerkey", "type": "user", "account_id": "557058:2e35028e-9ad8-4d6a-a3a5-275159a304a9"}, "title": "Client can crash leaving server still running", "component": null, "votes": 0, "watches": 1, "content": {"raw": "I'm usually testing by running `gazebo`.  If the client crashes, the next time I run `gazebo`, I get a segfault.  Then the next time I get\n\n        Exception [Master.cc:69] Unable to start server[Address already in use]\n\nThen I'm `killall`ing my way around, which is not pleasant.\n\nWe should clean this up.  I'm happy to help.", "markup": "markdown", "html": "<p>I'm usually testing by running <code>gazebo</code>.  If the client crashes, the next time I run <code>gazebo</code>, I get a segfault.  Then the next time I get</p>\n<div class=\"codehilite\"><pre><span></span>    Exception [Master.cc:69] Unable to start server[Address already in use]\n</pre></div>\n\n\n<p>Then I'm <code>killall</code>ing my way around, which is not pleasant.</p>\n<p>We should clean this up.  I'm happy to help.</p>", "type": "rendered"}, "assignee": {"display_name": "Nate Koenig", "uuid": "{c862cdd9-fcc8-4419-9fc7-e20db14b8fcb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D"}, "html": {"href": "https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:095b1e12-74ed-4e20-b44f-2f0745b616e0/ca24bb11-4787-4b14-a20d-d91835e9bde2/128"}}, "nickname": "Nathan Koenig", "type": "user", "account_id": "557058:095b1e12-74ed-4e20-b44f-2f0745b616e0"}, "state": "closed", "version": {"name": "all", "links": {"self": {"href": "data/repositories/osrf/gazebo/versions/225575.json"}}}, "edited_on": null, "created_on": "2012-10-09T23:47:38.021456+00:00", "milestone": null, "updated_on": "2016-09-14T19:45:02.989458+00:00", "type": "issue", "id": 38}